### ISSUE TITLE
Errors silently swallowed

### ISSUE THREAD

**Issue by jbq** [cbrand/gulp-protobufjs#5]
I'm using gulp-protobufjs to transform a proto file into JSON during build.  When there are compilation errors, they are silently swallowed, and noticing that the output JSON is missing is the only way to find out something was wrong.

Full example project provided: https://github.com/jbq/test-gulp-protobuf

Run `npm install` and `gulp`:

```
[09:45:28] Using gulpfile ~/protobuf/gulpfile.js
[09:45:28] Starting 'default'...
```

No output is generated.  No error is reported.

The error swallowed is: `Error: File not found: person.proto` I could get it by adding a `console.log()` in `node_modules/gulp-protobufjs/index.js` at line 76

**Comment by jbq** [comment]
Side note: I just tried `gulp-pbjs`: error reporting works, the code source looks simpler as it's a thin wrapper around the `pbjs` cli.  See my test at https://github.com/jbq/test-gulp-protobuf/tree/pbjs

**Comment by cbrand** [comment]
Hey thanks for the provided test. It helped me a lot. Good that there's another solution in it. 
The difference there and here seems to be that I am supporting streams and thus have to work with asynchronous code.

Gulp per default only seems to be ignoring error messages if they are passed asynchronously (even though the plugin basically "shouts" that there is an error thrown through events, error handling etc.).

Anyway I fixed it by manually printing the error to console and introducing a few more flags to be able to disable this or show the stack. I don't really think this is an optimal solution ( Documentation: https://github.com/cbrand/gulp-protobufjs/blob/1.3.0/README.md ).

Honestly I am rather dissatisfied with this solution. It really feels like a hack. But it is better than not showing any errors and leaving the user confused. I published Version 1.3.0 of the package in NPM. Your test is now showing an error on the console. Thanks again that you provided it!

If you have an alternative/better solution for it I will try to use it. However as of now I don't see a better workable way.
